backend/coins/btc/handlers/handlers.go

```diff
@@ -364,6 +364,52 @@ func (handlers *Handlers) getAccountBalance(*http.Request) (interface{}, error)
 	}, nil
 }
 
+type slip24Request struct {
+	RecipientName string `json:"recipientName"`
+	Nonce         string `json:"nonce"`
+	Memos         []struct {
+		Type         string `json:"type"`
+		Text         string `json:"text"`
+		Refund       string `json:"refund,omitempty"`
+		CoinPurchase struct {
+			CoinType int    `json:"coinType"`
+			Amount   string `json:"amount,omitempty"`
+			Address  string `json:"address,omitempty"`
+		} `json:"coinPurchase"`
```

Comment:
I would skip this as it's unused. By having it here it looks like it's needed.

Reply:
I already check for the memo type to be `text`. That should already cover it with a proper error message. I'll just remove the unused stuff.